app/controllers/katello/api/v2/errata_controller.rb

```diff
@@ -37,11 +37,11 @@ def available_for_content_view_filter(filter, collection)
     def custom_index_relation(collection)
       collection = filter_by_cve(params[:cve], collection) if params[:cve]
       if ::Foreman::Cast.to_bool(params[:errata_restrict_applicable])
-        collection = collection.applicable_to_systems(System.readable)
+        collection = collection.applicable_to_hosts(System.readable)
```

Comment:
I don't think you mean to pass System.readable to this method and the one below it.    We'll need something to look up readable hosts